src/views/ingresses/details/tabs/details/components/IngressRulesSection/RulesRows.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { FC } from 'react';
+
+import { IoK8sApiNetworkingV1IngressSpec } from '@kubevirt-ui/kubevirt-api/kubernetes/models/IoK8sApiNetworkingV1IngressSpec';
+import EmptyBox from '@utils/components/EmptyBox/EmptyBox';
+import { useNetworkingTranslation } from '@utils/hooks/useNetworkingTranslation';
+import { isEmpty } from '@utils/utils';
+import RulesRow from '@views/ingresses/details/tabs/details/components/IngressRulesSection/RulesRow';
+import { IngressPathRule } from '@views/ingresses/details/tabs/details/utils/types';
+import { getPort } from '@views/ingresses/details/tabs/details/utils/utils';
+
+type RulesRowsProps = {
+  namespace: string;
+  spec: IoK8sApiNetworkingV1IngressSpec;
+};
+
+const RulesRows: FC<RulesRowsProps> = ({ namespace, spec }) => {
+  const { t } = useNetworkingTranslation();
+
+  const rules: IngressPathRule[] = [];
+
+  if (spec?.rules) {
+    spec.rules.forEach((rule) => {
+      const paths = rule?.http?.paths;
+      if (isEmpty(paths)) {
+        rules.push({
+          host: rule.host || '*',
+          path: '*',
+          serviceName: spec?.defaultBackend?.service?.name,
+          servicePort: getPort(spec?.defaultBackend?.service),
+        });
+      } else {
+        paths.forEach((path) => {
+          rules.push({
+            host: rule.host || '*',
+            path: path.path || '*',
+            pathType: path.pathType,
+            serviceName: path?.backend?.service?.name,
+            servicePort: getPort(path?.backend?.service),
+          });
+        });
+      }
+    });
+
+    return (
+      <div className="co-m-table-grid__body">
+        {rules.map((rule) => {
+          // eslint-disable-next-line react/jsx-no-undef
```

Comment:
?

Reply:
I'm not sure. I removed it and it worked fine.